jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/VideoConstraints.kt

```diff
@@ -27,6 +27,10 @@ data class VideoConstraints(
         this["maxHeight"] = maxHeight
         this["maxFrameRate"] = maxFrameRate
     }.toJSONString()
+
+    companion object {
+        val ZERO = VideoConstraints(0)
```

Comment:
I wonder if there's a better name.  "zero" reads as "zero constraints" which sounds like 'unconstrained', but really it's more like "I want nothing at all", right?

Reply:
`ZERO_HEIGHT`?

Reply:
Or `NOTHING`?

Reply:
I'm not sure I have a good idea, it's hard to pair something with "constraints" that conveys "constrained in such a way that I know it will result in nothing at all".  `OFF`? `NOT_DISPLAYED`?

Reply:
I'm good with `NOTHING`